command_line/background.py

```diff
@@ -114,7 +121,7 @@ def run(args):
         pyplot.show()
 
 
-def background(imageset, indx, n_bins, mask_params=None):
+def background(imageset, indx, n_bins, corrected, mask_params=None):
```

Comment:
```suggestion
def background(imageset, indx, n_bins, corrected=False, mask_params=None):
```
I believe this is possibly the origin of the failing test (is called from here: https://github.com/dials/dials/blob/master/util/best.py#L13), and making this a defaulted parameter was something I was going to suggest anyway

Reply:
Thanks! I have done that - that does appear to be why the test fails - apologies for not checking before raising PR.

Reply:
That's what the CI system is for :)